### ISSUE TITLE
chart 1.0.0 not working on EKS

### ISSUE THREAD

**Issue by mvkrishna86** [grafana/pyroscope#2328]
#### Describe the bug
This chart 1.0.0 is giving below error on AWS EKS which is running on 1.23

`$ kp get hr
NAME        AGE     READY   STATUS
pyroscope   3d23h   False   Helm upgrade failed: chart requires kubeVersion: >=1.22 which is incompatible with Kubernetes v1.23.17-eks-2d98532
`
#### To Reproduce

1. Deploy pyroscope helm chart in EKS 1.23

#### Expected behavior

It should be deployed without any issues.

#### Environment

- Infrastructure: Kubernetes
- Deployment tool: helm

**Comment by cyriltovena** [comment]
👋 which version of helm are you using ? I think I had the same issue while testing the chart and I had to update my helm binary.

**Comment by mvkrishna86** [comment]
I am using 1.0.0 (latest one)

**Comment by cyriltovena** [comment]
Can you try 1.0.1 helm chart I just release ?

```
helm repo update
helm search repo grafana/pyroscope --versions
NAME             	CHART VERSION	APP VERSION	DESCRIPTION                           
grafana/pyroscope	1.0.1        	1.0.0      	🔥 horizontally-scalable, highly-available, mul...
grafana/pyroscope	1.0.0        	1.0.0      	🔥 horizontally-scalable, highly-available, mul...
grafana/pyroscope	0.9.0        	1.0.0-rc.0 	🔥 horizontally-scalable, highly-available, mul...
```

**Comment by cyriltovena** [comment]
Should be fixed in 1.0.1 let me know if not.

**Comment by mvkrishna86** [comment]
I am able to deploy the chart with 1.0.1
